app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt

```diff
@@ -196,10 +199,11 @@ class ProfileFragment : Fragment(), ProfileView, ActionMode.Callback {
 
             READ_STORAGE_REQUEST_CODE -> {
                 if (resultCode == RESULT_OK) {
-                    avatarImage = File(data!!.data.getRealPathFromURI(context!!))
+                    avatarImageUri = data!!.data
```

Comment:
You need to check if the data intent isn't null after the `resultCode == RESULT_OK`

```
if (resultCode == RESULT_OK) {
 if (data != null) {
  [...]
 }
}
```